src/main/java/org/apache/maven/shared/utils/io/FileUtils.java

```diff
@@ -109,7 +114,7 @@ protected FileUtils()
     /**
      * The file copy buffer size (30 MB)
      */
-    private static final long FILE_COPY_BUFFER_SIZE = ONE_MB * 30;
+    private static final int FILE_COPY_BUFFER_SIZE = ONE_MB * 30;
```

Comment:
Don't think there was any need for this to be a `long` other than the only previous use case, where it's now automatically cast to a long anyway.